### PR TITLE
[link] Allow both router links and regular links to co-exist in same document

### PR DESCRIPTION
**Pull request by tmorehouse**
In some cases one would like to have both regular links (i.e. links to an in-page ID) and router links (routed by vue-router).

If `to` is not specified, then assume a regular local link (not handled by router).

Addresses #362 and #363  (I think... discussion is welcome)

### CODE REVIEW

File: lib/components/link.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
     <!-- When VueRouter is available -->
-    <a v-if="routerAvailable"
+    <a v-if="routerAvailable && to"
```
Comment (pi0) [inline lib/components/link.vue]: Can't we refactor `&& to` into computed property?

File: lib/components/link.vue [hunk 2]
```diff
@@ -52,16 +52,12 @@
             },
             _to() {
                 if (!this.routerAvailable || this.disabled) {
```
Comment (pi0) [inline lib/components/link.vue]: Then this condition won't happen  as `routerAvailable` is false when no `to` prop exists :))

**Comment by tmorehouse** [comment]
@pi0, check the new changes and see what you think?

**Comment by pi0** [comment]
LGTM :)

**Comment by tmorehouse** [comment]
Hopefully this doesn't break something else :P

**Comment by pi0** [comment]
Something is breaking Nav links. (Documentation website is broken)

**Comment by tmorehouse** [comment]
Is documentation site using latest release or master?

**Comment by pi0** [comment]
Testing locally on master. I think condition should be `return this.$router && this.to` :))

**Comment by tmorehouse** [comment]
Agh, lol..  an easy fix (I hope)

**Comment by pi0** [comment]
Yep, i'll publish fix soon

**Comment by MarZab** [comment]
Shouldn't in the case neither href or to provided, the link be rendered as `<a role="button>"` since bootstrap does support this troughout? Related to: https://github.com/bootstrap-vue/bootstrap-vue/issues/362

**Comment by tmorehouse** [comment]
So in the case of a `dropdown-item`, having both HREF and TO blank should render a `button.dropdown-item` instead of a link.

**Comment by tmorehouse** [comment]
BS V4.alpha.6 `.dropdown-items` support both using links or buttons (although there are slight styling issues between the two font-wise).  Maybe the dropdown-item component should create a `<button class="dropdown-item>` when both HREF and TO are falsy? rather than using a link tag?.

**Pull request merged**